src/integrations/faq.md

```diff
@@ -127,6 +127,37 @@ We use the [Gravity Bridge](https://github.com/Gravity-Bridge) blockchain to bri
 
 If you need to transfer tokens across the bridge, we recommend users use Cosmostation's [spacestation.zone](https://spacestation.zone/) dApp with Metamask and Keplr. 
 
+### What is Circulating Supply and how to find out the distribution amount?
+
+Circulating supply is the total number of available `NYM`. `NYM` is currently present on the IBC-enabled Nyx blockchain, as well as in ERC20 form on Ethereum Mainnet.
+
+The Validator API endpoints can be found via the [Swagger Documentation](https://validator.nymtech.net/api/swagger/index.html). The following endpoints can be called to retrieve the correct distribution amount and circulating supply within Nym. 
+
+Using this API endpoint returns information about the circulating supply of Nym tokens:
+
+```
```

Comment:
I don't understand the formatting here: how do the bullet points relate to this code block?

Reply:
Not very clear - are the bullet points fields that are  returned by that querying that single API endpoint?